src/zenml/integrations/bentoml/steps/bentoml_deployer.py

```diff
@@ -102,40 +116,62 @@ def service_apis(bento_tag: str) -> List[str]:
         apis_paths = list(apis.keys())
         return apis_paths
 
-    # create a config for the new model service
-    predictor_cfg = BentoMLDeploymentConfig(
-        model_name=model_name,
-        bento=str(bento.tag),
-        model_uri=bento.info.labels.get("model_uri"),
-        bento_uri=bento.info.labels.get("bento_uri"),
-        apis=service_apis(str(bento.tag)),
-        workers=workers,
-        host=host,
-        backlog=backlog,
-        working_dir=working_dir or source_utils.get_source_root(),
-        port=port,
-        pipeline_name=pipeline_name,
-        pipeline_step_name=step_name,
-        ssl_parameters=SSLBentoMLParametersConfig(
-            ssl_certfile=ssl_certfile,
-            ssl_keyfile=ssl_keyfile,
-            ssl_keyfile_password=ssl_keyfile_password,
-            ssl_version=ssl_version,
-            ssl_cert_reqs=ssl_cert_reqs,
-            ssl_ca_certs=ssl_ca_certs,
-            ssl_ciphers=ssl_ciphers,
-        ),
-    )
-
+    def create_deployment_config(
+        deployment_type: Literal["local", "container"],
+    ) -> Tuple[ServiceConfig, ServiceType]:
+        common_config = {
+            "model_name": model_name,
+            "bento_tag": str(bento.tag),
+            "model_uri": bento.info.labels.get("model_uri"),
+            "bento_uri": bento.info.labels.get("bento_uri"),
+            "apis": service_apis(str(bento.tag)),
+            "host": host,
+            "port": port,
+            "pipeline_name": pipeline_name,
+            "pipeline_step_name": step_name,
+            "workers": workers,
+            "backlog": backlog,
+        }
+
+        if deployment_type == "container":
```

Comment:
can we create an enum for this and have both local and container maybe at some point we add k8s

Reply:
done!